test/corecel/grid/NonuniformGrid.test.cc

```diff
@@ -64,6 +64,26 @@ TEST_F(NonuniformGridTest, find)
     EXPECT_THROW(grid.find(10), DebugError);
 #endif
 }
+
+TEST_F(NonuniformGridTest, find_edge)
+{
+    GridT grid(irange, ref);
+    using result_type = GridT::result_type;
+
+#if CELERITAS_DEBUG
+    EXPECT_THROW(grid.find(-1), DebugError);
+#endif
```

Comment:
Use C++ `if` instead of C preprocessor macros unless the embedded code is invalid (e.g. calling an external library that isn't `#include`d).
```suggestion
    if (CELERITAS_DEBUG)
    {
        EXPECT_THROW(grid.find(-1), DebugError);
    }
```